website/docs/cloud-docs/run/cli.mdx

```diff
@@ -194,16 +194,16 @@ Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
 
 ### Non-Interactive Workflows
 
-> **Hands On:** Try the [Deploy Infrastructure with Terraform Cloud and CircleCI](https://learn.hashicorp.com/tutorials/terraform/circle-ci) tutorial on HashiCorp Learn.
+> **Hands On:** Try the [Deploy Infrastructure with Terraform Cloud and CircleCI](https://learn.hashicorp.com/tutorials/terraform/circle-ci) tutorial.
 
 External systems cannot run the traditional apply workflow because Terraform requires console input from the user to approve plans. We recommend using the [API-driven Run Workflow](/cloud-docs/run/api) for non-interactive workflows when possible.
 
 If you prefer to use the CLI in a non-interactive environment, we recommend first running a [speculative plan](/cloud-docs/run#speculative-plans) to preview the changes Terraform will make to your infrastructure. Then, use one of the following approaches with the `-auto-approve` flag based on the [execution mode](/cloud-docs/workspaces/settings#execution-mode) of your workspace. The [`-auto-approve`](/cli/commands/apply#auto-approve) flag skips prompting you to approve the plan.
```

Comment:
okay in this ONE case, I genuinely think we leave the will. Hear me out - essentially, we're telling the user that the speculative plan lets you review what terraform WILL Do - like genuinely what it is planning to do in the future. 

I was thinking of alternatives:
- to preview the changes Terraform plans to make to your infrastructure (eh, I don't like this one because it uses plans again)
- to preview the changes Terraform is going to make (is this even better lol - it's a will substitute) 

and "makes" just doesn't feel quite right here as it's genuinely a future action. Do you have other ideas? This is just one place I wanted to come back to because it's kind of an odd one regarding will.